components/script/dom/bindings/trace.rs

```diff
@@ -160,12 +160,6 @@ impl<T: JSTraceable> JSTraceable for Rc<T> {
     }
 }
 
-impl JSTraceable for JobQueue {
-    fn trace(&self, trc: *mut JSTracer) {
-        self.0.trace(trc)
-    }
-}
-
 impl JSTraceable for Job {
```

Comment:
Oh, wait. I can auto derive. The type on which error was thrown was `JobType`. Updated.